### PR TITLE
Improved styling

### PR DESCRIPTION
**Pull request by JoelForamitti**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -23,19 +27,47 @@ const { user } = storeToRefs(userStore);
...
+/* Global styles for primevue components */
+[type="checkbox"],
+[type="radio"] {
+  border: 1px solid #676d73;
+}
 .p-checkbox.p-component {
   display: flex;
 }
+
+/* TODO: Tabview full height as a component */
```
Comment (nanu-c) [inline src/App.vue:54]: If you use scss this should be like this
```
.tabview-full-height {
    .p-tabview {
        height: 100%;
        display: flex;
        flex-direction: column;
    }

    .p-tabview-nav-container {
        box-shadow: 0 0px 1px -1px rgba(0, 0, 0, 0.2), 0 0px 1px 0 rgba(0, 0, 0, 0.14), 0 0px 3px 0 rgba(0, 0, 0, 0.12);
    }
    .p-tabview-panels,
    .p-tabview-panel {
        height: 100%;
        overflow: auto;
        background: none !important;
        padding: 10px 0px 0px 0px !important;
    }
}

```

File: src/components/SidebarMenu.vue [hunk 2]
```diff
@@ -44,74 +46,65 @@
...
+    </PanelMenu>
+  </div>
 </template>
 
 <script lang="ts" setup>
 import { useMenuStore } from "@/stores/menu";
 import { useUserStore } from "@/stores/user";
 import { storeToRefs } from "pinia";
-import PrimePanelMenu from "primevue/panelmenu";
+import PanelMenu from "primevue/panelmenu";
```
Comment (nanu-c) [inline src/components/SidebarMenu.vue:104]: This change is against our convention to name all prime items as Prime

File: src/components/SidebarMenu.vue [hunk 3]
```diff
@@ -181,43 +175,30 @@ watch(mainMenu, () => {
...
-  padding-bottom: 4px;
 }
-.p-panelmenu .p-panelmenu-header .p-panelmenu-header-content {
-  border-radius: 0px !important;
+
+.sidebar_menu a {
+  text-decoration: none;
+  font-weight: 100;
+  font-size: 1rem;
+  color: rgb(73, 80, 87);
```
Comment (nanu-c) [inline src/components/SidebarMenu.vue:202]:
```
.sidebar_menu {
    border: none;
    width: 100%;
    &.p-panelmenu {
        .p-panelmenu-panel {
            margin: 5px;
        }
        .p-panelmenu-header .p-panelmenu-header-content {
            border-radius: 5px !important;
            background: none;
            border: none;
        }
        .p-menuitem-text {
            padding-top: 4px;
        }
    }

    a {
        text-decoration: none;
        font-weight: 100;
        font-size: 1rem;
        color: #495057;
    }
}
```
And also colors should be used as hex

File: src/components/datatable/ObjectTable.vue [hunk 4]
```diff
@@ -113,11 +113,12 @@ watch(
   },
   { immediate: true }
 );
-
+console.log(filters.value);
```
Comment (nanu-c) [inline src/components/datatable/ObjectTable.vue:116]: remove :)

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (JoelForamitti) [reply src/App.vue:54]: done! much better like this

File: src/components/SidebarMenu.vue (hunk 2 above)
Comment (JoelForamitti) [reply src/components/SidebarMenu.vue:104]: done! thanks for spotting

File: src/components/SidebarMenu.vue [hunk 5]
```diff
@@ -165,6 +158,7 @@ function buildMenu(menu: any, items: any) {
       items.value[0].items.push(item);
     }
   }
+  console.log(items.value);
```
[the remark this answers is not in the document]
Comment (JoelForamitti) [reply src/components/SidebarMenu.vue:161]: done! thanks for spotting

File: src/components/datatable/ObjectTable.vue (hunk 4 above)
Comment (JoelForamitti) [reply src/components/datatable/ObjectTable.vue:116]: done! sorry ..

**Pull request merged**